pkg/anago/stage.go

```diff
@@ -817,3 +861,142 @@ func (d *DefaultStage) StageArtifacts() error {
 	)
 	return nil
 }
+
+// GenerateAttestation creates a provenance attestation with its predicate
+// preloaded with the current krel run information
+func (d *defaultStageImpl) GenerateAttestation(state *StageState, options *StageOptions) (attestation *provenance.Statement, err error) {
+	// Build the arguments RawMessage:
+	arguments := map[string]string{
+		"--type=":          options.ReleaseType,
+		"--branch=":        options.ReleaseBranch,
+		"--build-version=": options.BuildVersion,
```

Comment:
Non-blocking:

```suggestion
		"release-type":          options.ReleaseType,
		"branch":        options.ReleaseBranch,
		"build-version": options.BuildVersion,
```

Otherwise, we get:

```json
    "recipe": {
      "type": "https://cloudbuild.googleapis.com/CloudBuildYaml@v1",
      "definedInMaterial": 0,
      "entryPoint": "https://github.com/kubernetes/release/blob/master/gcb/stage/cloudbuild.yaml",
      "arguments": {
        "--branch=": "master",
        "--build-version=": "v1.23.0-alpha.3.105+0ac956ff2bef9c",
        "--type=": "alpha"
      }
    },
```

(Note the extraneous `--` and `=` in the arguments.)